### DISCUSSION TITLE
Nothing displayed on my browser

### DISCUSSION THREAD

**Discussion by martataddese** [Aklilu-Mandefro/askMe#3] [Q&A] [answered]
Hello dear folks, when I run the app, nothing is displayed on my browser. Can you please explain what I made wrong?

Thanks.

**Comment by Aklilu-Mandefro** [comment, marked as answer]
Hello dear Marta, to run the **askMe** app, you can't run the app by just opening the root **index.html** file in your browser. But, you have to run it, either:

1.  In a local computer with xampp or wampserver with PHP 7 or higher. Or
2. In any live environment like Vercel.

**Reply by martataddese** [reply]
Thanks a ton, now it worked.
